Lesson_16/l16_h01_class.py

```diff
@@ -0,0 +1,77 @@
+"""
```

Comment:
Good start, but there should be added card operations . For example : pay,  issue card, block etc.
In other words you have provided class attributes but was expected to see class methods too ))